### ISSUE TITLE
URLs skipped due to if_not_archived_within option treated as failures

### ISSUE THREAD

**Issue by AnyOldName3** [overcast07/wayback-machine-spn-scripts#33]
I'm trying to archive the missing parts of the website for an open-source library so it can be replaced by a new one without losing any information to history. As I know much of the site is already on the Wayback Machine, and that it's not been changed recently, I've used the `if_not_archived_within` option with a large value to avoid unnecessarily archiving another copy of things that don't need it.

`spn.sh` is treating the responses it gets back, e.g. `The same snapshot had been made n hours, n minutes ago. You can make new capture of this URL after n hours.`, as failures, but as far as I'm concerned, this is a success - the page is in the archive.

This isn't a massive problem on its own, as I could just let the script run on my URL list, but I've had a series of failures, and had to use the resume feature, and each time I do, the script decides it must attempt all these URLs again, wasting a bunch of time, and increasing the chances that another failure will knock it out part way again.

I don't really mind whether it's a flag or the default behaviour, but it'd be preferable if the *same snapshot* response was considered a success.

**Comment by overcast07** [comment]
Yeah, I usually do something like that myself if I have to remove URLs that resulted in failed captures from the URL list, although it's not ideal since the script is theoretically not supposed to require much technical competence to use.
